src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
```

Comment:
Consider implementing a private constructor to prevent instantiation and ensure the encapsulation principle is followed, especially if this class is only meant to be used as a data holder.